prog/vm/host_nexus.rb

```diff
@@ -105,6 +110,33 @@ def before_run
     }
   end
 
+  def default_boot_image_version(image_name)
+    case image_name
+    when "ubuntu-jammy"
+      Config.ubuntu_jammy_version
+    when "github-ubuntu-2204"
+      Config.github_ubuntu_2204_version
+    when "github-ubuntu-2004"
+      Config.github_ubuntu_2004_version
+    when "github-gpu-ubuntu-2204"
+      Config.github_gpu_ubuntu_2204_version
+    when "postgres-ubuntu-2204"
+      Config.postgres_ubuntu_2204_version
+    else
+      fail "Unknown boot image: #{image_name}"
+    end
```

Comment:
```suggestion
    fail "Unknown boot image: #{image_name}" unless Config.respond_to?(:"#{image_name.tr("-", "_")}_version")
    Config.send(:"#{image_name.tr("-", "_")}_version") : nil
```

Reply:
No. Let's not do this. It just isn't clear.

Reply:
I agree. I am not a fan of magic.